src/components/ReportActionItem/MoneyReportView.tsx

```diff
@@ -76,6 +77,22 @@ function MoneyReportView({report, policy, isCombinedReport = false, shouldShowTo
     const shouldShowReportField =
         !ReportUtils.isClosedExpenseReportWithNoExpenses(report) && ReportUtils.isPaidGroupPolicyExpenseReport(report) && (!isCombinedReport || !isOnlyTitleFieldEnabled);
 
+    const renderThreadDivider = useMemo(
+        () =>
+            shouldHideThreadDividerLine && !isCombinedReport ? (
+                <UnreadActionIndicator
+                    reportActionID={report.reportID}
+                    shouldHideThreadDividerLine={shouldHideThreadDividerLine}
+                />
+            ) : (
+                <SpacerView
+                    shouldShow={!shouldHideThreadDividerLine}
+                    style={[!shouldHideThreadDividerLine ? styles.reportHorizontalRule : {}]}
+                />
+            ),
+        [shouldHideThreadDividerLine, report.reportID, styles.reportHorizontalRule, isCombinedReport],
```

Comment:
Why is `styles.reportHorizontalRule` included here? That will never change based on onyx data, right? 🤔 

```suggestion
        [shouldHideThreadDividerLine, report.reportID, styles.reportHorizontalRule, isCombinedReport],
```

Reply:
@roryabraham set me straight

The answer is not "we use this variable inside `useMemo(` so it needs to be in the dependency array"

It's more like styles are dynamic / can be recalculated based on the theme - and since they're dynamic and could be changed, we should include them in the dependency array